.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Summary
+
+*Briefly describe the issue and please DELETE the non applicable headings below. *
+
+## System information
+
+*Please specify the Kolibri version you were using and your operating system!*
+
+ - Version: ?
+ - Operating system: ?
+ - Browser: ?
+
+## Traceback or relevant snippet from server.log
+
+```
+INSERT TRACEBACK, LOG MESSAGES ETC. HERE
+```
+
+## How to reproduce
+
+1. Steps
+1. To
+1. Reproduce
+
+## Screenshots
+
+*If applicable*
+
+## Real-life consequences (anything community should be aware, for instance how it affects your deployment)
```

Comment:
Modify to:
`## Real-life consequences (anything community should be aware of, for instance how it affects your deployment)`